filters/glitch/src/GlitchFilter.ts

```diff
@@ -106,7 +106,9 @@ class GlitchFilter extends Filter
 
     private _offsets: Float32Array = new Float32Array(1);
     private _sizes: Float32Array = new Float32Array(1);
-    private _direction = 0;
+
+    /** direction is actually a setter for uniform.cosDir and uniform.sinDir. Initializing it prevents those values from being written! */
+    private _direction: number;
```

Comment:
Maybe lets do this to satisfy the compiler
```suggestion
    private _direction = -1;
```

Reply:
was my first idea but felt that undefined was the "correct" fix 😛